src/Import/include/FMI3/fmi3_import_variable.h

```diff
@@ -84,6 +91,17 @@ typedef struct fmi3_xml_clock_variable_t fmi3_import_clock_variable_t;
 typedef struct fmi3_import_variable_list_t fmi3_import_variable_list_t;
 /**@} */
 
+/**
+ * @name Types encapsulating alias information
+ * @{
+ */
+/** \brief List of the alias variables for a non-alias variable */
+typedef struct fmi3_xml_alias_variables_t fmi3_import_alias_variables_t;
```

Comment:
Maybe a more distinct name such as `fmi3_xml_alias_variables_list_t `?
Too easy to mix up with `fmi3_xml_alias_variable_t `

Reply:
Arrays, lists, and what not are troublesome when part of the API. While I would prefer `_list_t` I decided to not do it due to conventions.

```
FMILIB_EXPORT fmi3_import_unit_definitions_t* fmi3_import_get_unit_definitions(fmi3_import_t* fmu);
FMILIB_EXPORT fmi3_import_type_definitions_t* fmi3_import_get_type_definitions(fmi3_import_t* fmu);
```
And then we have more advanced lists for variables. See `fmi3_import_variable_list.h`. They have suffix `_list_t` so didn't want to mix them up.

But I would be happy with renaming to list if we could rename `fmi3_import_get_type_definitions` etc. (and also fix dimensions vector to be similar).

Reply:
I'd be happy to rename all lists to `*_list_t` for consistency, probably in a follow up issue. 

I cannot really of another way to distinguish it by naming otherwise right now, without breaking the consistency with respect to `fmi3_xml_alias_variable_t`.